docker-compose.yml

```diff
@@ -3,7 +3,8 @@ services:
     build:
       context: ./server
       args:
-        SECRET_KEY: secret
+        - SECRET_KEY=secret
```

Comment:
can you add a comment saying `Change this!`?

Reply:
also missing `JWT_SECRET`

Reply:
should all be resolved here: https://github.com/MaxLeiter/Drift/pull/75/commits/b3c99a88510319216de04a00626d953af33efc37